min-max/maxMin.js

```diff
@@ -0,0 +1,42 @@
+var arr = [5,10,45,100,80,7,9,108,1,500];
+
+var value = getMaxOrMin(arr,'min');
+console.log(value);
+
+
+function getMaxOrMin(arr,option) {
+    var maxMin = option.toUpperCase();
+    switch(maxMin){
+        case 'MAX': 
+        value = getMax(arr);
+        break;
+        case 'MIN':
+        value = getMin(arr);
+        break;
+        default:
+        value = "Invalid option. Provider 'MIN' or 'MAX'";
```

Comment:
Typo: 'provider' -> 'provide'

Reply:
We always work with provider that's why mistakenly.